waspc/e2e-test/GoldenTest.hs

```diff
@@ -79,27 +80,52 @@ runGoldenTest goldenTest = do
           goldenOutputAbsFp
           currentOutputAbsFp
           (return ()) -- A no-op command that normally generates the file under test, but we did that in bulk above.
-        | currentOutputAbsFp <- manifestAbsFp : currentOutputAbsFps,
+        | currentOutputAbsFp <- manifestAbsFp : filesForGoldenTestAbsFps,
           let goldenOutputAbsFp = remapCurrentPathToGolden currentOutputAbsFp
       ]
   where
-    listRelevantTestOutputFiles :: FilePath -> IO [FilePath]
-    listRelevantTestOutputFiles dirToFilterAbsFp =
-      getDirFiltered (return <$> isTestOutputFileTestable) dirToFilterAbsFp >>= filterM doesFileExist
-
-    isTestOutputFileTestable :: FilePath -> Bool
-    isTestOutputFileTestable fp =
+    manifestFileName :: String
+    manifestFileName = "files.manifest"
+
+    -- Lists files that should be included in the `files.manifest`.
+    listFilesForManifest :: FilePath -> IO [FilePath]
+    listFilesForManifest dirToFilterAbsFp =
+      getDirFiltered (return <$> shouldIncludeInManifest) dirToFilterAbsFp >>= filterM doesFileExist
+
+    -- Lists files that should be compared in golden tests.
+    listFilesForGoldenTest :: FilePath -> IO [FilePath]
+    listFilesForGoldenTest dirToFilterAbsFp =
```

Comment:
Similar to my other comment: golden tests are a non-standard implementation name. A better name for this variable is `getFilesForCheckingContents` or something like that.

Same thing about `listFilesForManifest`.

I think we should adopt this naming scheme to the rest of the code too.